openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/CEEnvironmentProcessor.java

```diff
@@ -106,11 +117,14 @@ public void createOpenShiftResource(@Observes(precedence = 10) Before event, Ope
 
         log.info(String.format("Creating environment for %s method %s", testClass.getName(), testMethod));
 
-        OpenShiftResourceFactory.createResources(testClass.getJavaClass(), client, testMethod, cubeOpenShiftConfiguration.getProperties());
+        if (cubeOpenShiftConfiguration.isNamespaceMethodScopeEnabled()) {
+            createUniqueNamespace(cubeOpenShiftConfiguration, testMethod.getName());
```

Comment:
maybe in case of method should be something like `testClass.getSimpleName() + "_" + testMethod.getName()`